lib/Internal/Controls.js

```diff
@@ -467,8 +467,8 @@ export default class Controls extends CoreBase {
 					CHOICES_BUTTON,
 					{
 						type: 'checkbox',
-						label: 'Treat stepped as pressed? (latch compatability)',
-						id: 'latch_compatability',
+						label: 'Treat stepped as pressed? (latch compatibility)',
+						id: 'latch_compatibility',
```

Comment:
```suggestion
						id: 'latch_compatability',
```

Renaming this will break any existing actions for users.